manifest/spark-operator-with-metrics.yaml

```diff
@@ -21,13 +21,13 @@ metadata:
   namespace: spark-operator
   labels:
     app.kubernetes.io/name: sparkoperator
-    app.kubernetes.io/version: v1beta2-1.2.0-3.0.0
+    app.kubernetes.io/version: v1beta2-1.2.3-3.1.1
```

Comment:
`v1beta2-1.2.1-3.1.1`?

Reply:
Same, There was already release [v1beta2-1.2.2](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/releases/tag/v1beta2-1.2.2-3.0.0) so next one should be v1beta2-1.2.3

Reply:
https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/releases/tag/v1beta2-1.2.2-3.0.0

Reply:
Yep, `v1beta2-1.2.3-3.1.1` is the right one.